tests/integration/api/test_sc_test_jobs.py

```diff
@@ -0,0 +1,34 @@
+from tests.base import BaseTest
+
+from tenable_io.api.models import ScTestJob
+
+
+class TestScTestJobsApi(BaseTest):
+
+    def test_get(self, client, image):
+        containers = client.sc_containers_api.list()
+        assert len(containers) > 0, u'At least one image exists.'
+        container = [c for c in containers if image['name'].endswith(c.name) and
+                     c.digest == u'sha256:%s' % image['digest']][0]
+        assert container.name == image['name'], \
+            u'Image returned from the container list should match the image that was uploaded.'
```

Comment:
Did you get the above from the containers api list test? You probably don't need to test those again. I think what you have below is sufficient, however, you should still inject `image` to ensure that there is at least one job.